google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -169,12 +170,27 @@ def execute(self, operation, parameters=None, job_id=None, job_config=None):
         formatted_operation = _format_operation(operation, parameters=parameters)
         query_parameters = _helpers.to_query_parameters(parameters)
 
-        config = job_config or job.QueryJobConfig(use_legacy_sql=False)
+        if client._default_query_job_config:
+            if job_config:
+                config = job_config._fill_from_default(client._default_query_job_config)
+            else:
+                config = copy.deepcopy(client._default_query_job_config)
+        else:
+            config = job_config or job.QueryJobConfig(use_legacy_sql=False)
+
         config.query_parameters = query_parameters
         self._query_job = client.query(
             formatted_operation, job_config=config, job_id=job_id
         )
 
+        if self._query_job.dry_run:
+            schema_field = schema.SchemaField(
+                name="estimated_bytes", field_type="INTEGER", mode="REQUIRED",
```

Comment:
Done.

I set the description (i.e. schema) to `None`, because there is no `schema` property neither on the `QueryJob`, nor on the `QueryJobConfig` instances.